Source/Models/FORMFieldValidation.h

```diff
@@ -4,10 +4,12 @@
 @interface FORMFieldValidation : NSObject
 
 @property (nonatomic, getter = isRequired) BOOL required;
-@property (nonatomic) NSUInteger minimumLength;
-@property (nonatomic) NSInteger maximumLength;
-@property (nonatomic) NSString *format;
-@property (nonatomic) CGFloat minimumValue;
-@property (nonatomic) CGFloat maximumValue;
+@property (nonatomic, strong) NSNumber *minimumLength;
```

Comment:
I guess we haven't done that in the app yet, but I made an issue for it!

https://github.com/hyperoslo/Form/issues/230